privx_api/authorizer.py

```diff
@@ -557,3 +560,28 @@ def search_certificates(
             body=cert_params,
         )
         return PrivXAPIResponse(response_status, HTTPStatus.OK, data)
+
+    def get_certificates_list(self) -> PrivXAPIResponse:
+        """
+        Get all Certificates.
+
+        Returns:
+            PrivXStreamResponse
+        """
+        response_status, data = self._http_get(
+            UrlEnum.AUTHORIZER.GET_CERTIFICATES_LIST,
+        )
+        return PrivXAPIResponse(response_status, HTTPStatus.OK, data)
+
+    def get_certificate_by_id(self, cert_id: str) -> PrivXAPIResponse:
+        """
+        Get Certificate by ID
+
+        Returns:
+            PrivXStreamResponse
```

Comment:
Such func returns PrivXAPIResponse, not a PrivXStreamResponse

Reply:
@AlicxSSH let's fix the remaining issues

Reply:
i just need to fix the comments? the rest is ok?